integration_tests/run_pyspark_from_build.sh

```diff
@@ -102,7 +102,7 @@ else
     RUN_TESTS_COMMAND=("$SCRIPTPATH"/runtests.py
       --rootdir
       "$LOCAL_ROOTDIR"
-      "$LOCAL_ROOTDIR"/src/main/python)
+      "$LOCAL_ROOTDIR"/src/main/python/orc_test.py)
```

Comment:
Why was this change made?

Reply:
Nice catch and updated.
Uploaded this by mistake.